tests/utils/TestCase/DominantColorTestCase.php

```diff
@@ -24,7 +24,7 @@ public function provider_get_dominant_color() {
 			),
 			'green_jpg'     => array(
 				'image_path'            => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/green.jpg',
-				'expected_color'        => array( '00ff00', '00ff01' ),
+				'expected_color'        => array( '00ff00', '00ff01', '02ff01' ),
```

Comment:
Why is this in here? Seems unrelated? If there's another PR that caused breakage here, I think we should fix this separately.

Reply:
Actually, some of the dominant color tests were checking this green color, but with quality changes, it seems that `02ff01` is another green variant being created in tests for the jpg images.